future-apps/payroll/app/src/components/Badge/Badge.Identity.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
```

Comment:
Shouldn't we call this file Badge.Transaction?

Reply:
Makes sense to me